README.md

```diff
@@ -50,6 +50,32 @@ npm run build
 - Twitter: [@iamdeewyne](https://twitter.com/iamdeewyne)
 - LinkedIn: [@daniel-yerimah](https://www.linkedin.com/in/daniel-yerimah/)
 
+👤 **Benjamin Semah**
+
+- GitHub: [Benjamin Semah](https://github.com/BenjaminSemah)
+- Twitter: [Benjamin Semah](https://twitter.com/BenjaminSemah)
+- LinkedIn: [Benjamin Semah](https://www.linkedin.com/in/benjaminsemah/)
+
+👤 **Bertrand Mutangana Ishimwe**
+
+- GitHub: [Bertrand Mutangana Ishimwe](https://github.com/BertrandConxy)
+- Twitter: [Bertrand Mutangana Ishimwe](https://twitter.com/BertrandMutanga)
+- LinkedIn: [Bertrand Mutangana Ishimwe](https://www.linkedin.com/in/bertrand-mutangana-024905220/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Yerimah/JS-Capstone-Project/issues).
+
 ## Show your support
 
-Give a ⭐️ if you like this project!
\ No newline at end of file
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse for the [involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270
+)
+- TV Maze for the [Tv Maze API](https://www.tvmaze.com/api
+) 
```

Comment:
- [ ] The readme file is missing the license and 'run tests' sections, as required on the [Microverse template](https://github.com/microverseinc/readme-template)


Please, ensure that your readme file contains all the required missing sections.

Reply:
Resolved ✔️